internal/model/todos.go

```diff
@@ -10,43 +14,42 @@ type Todo struct {
 
 // TodoList kind of a has many struct
 type TodoList struct {
-	ID    string `json:"id"`
-	Name  string `json:"name"`
-	Todos []Todo `json:"todos"`
+	ID    string  `json:"id"`
+	Name  string  `json:"name"`
+	Todos []*Todo `json:"todos"`
```

Comment:
Almost always want to store "pointers" to structs and not copies. Anything in the struct gets copied when you're passing it around, this can be expensive. Using the "*" says that I store a pointer to an existing struct, and when you pass it around you're sending it's address of where it lives. So this becomes a slice of addresses to Todos.